### PR TITLE
Views/17 contact desktop raulmarty

### PR DESCRIPTION
**Pull request by MartyMaker**

### CODE REVIEW

File: src/styl/fonts.styl [hunk 1]
```diff
@@ -29,6 +29,16 @@
 }
 
 @font-face {
+  font-family: 'Roboto-Medium';
+  src: url('../fonts/Roboto-Medium.eot?#iefix') format('embedded-opentype'),
+  url('../fonts/Roboto-Medium.woff') format('woff'),
```
Comment (ZhavaMan) [inline src/styl/fonts.styl]: please, indent source url's

File: src/styl/fonts.styl [hunk 2]
```diff
@@ -29,6 +29,16 @@
 }
 
 @font-face {
+  font-family: 'Roboto-Medium';
```
Comment (ZhavaMan) [inline src/styl/fonts.styl]: DO NOT specify font weight in family name

File: src/styl/fonts.styl [hunk 3]
```diff
@@ -91,6 +101,10 @@
   font-family: 'Roboto-Bold';
 }
 
+.rer-font-roboto-medium {
+  font-family: 'Roboto-Medium';
```
Comment (ZhavaMan) [inline src/styl/fonts.styl]: specify font weight in font **class**

Comment (ZhavaMan) [inline src/styl/fonts.styl]: Do not specify font weight in font family

File: src/styl/sections/contact.styl [hunk 4]
```diff
@@ -0,0 +1,135 @@
+.contact-container
+  background url('../img/backgrounds/solar-benefits-2.png')
+  background-position center
+  background-repeat no-repeat
+  background-size cover
+
+.contact-tite
+  color #132c3e
+  font-size remify(35px)
+  font-weight $regular
```
Comment (ZhavaMan) [inline src/styl/sections/contact.styl]: specify font weight in font *class*
line 105

File: src/templates/sections/contact.njk [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+{% block content %}
+  <div class="contact-container">
+    <h1 class="contact-tite rer-font-roboto-regular">Contacto</h1>
+    <div class="contact-content">
+      <form class="contact-form">
+        <div class="contact-form-content">
+          <div class="contact-input-content margin-right-59px">
+            <label class="contact-input-label rer-font-roboto-medium" for="contact-name">Nombre</label>
+            <input class="contact-input" type="" name="" id="contact-name">
```
Comment (jresendiz27) [inline src/templates/sections/contact.njk]: Do the input tags need to have the `name` and `type` attribute  @ZhavaMan ? It's just a question.

**Pull request merged**